Description: inconsistent order of #include causes incompatible struct types
 While building the package using our research compiler infrastructure we
 noticed incompatible types of struct re_pattern_buffer (from regex.h) being
 used within the declaration of GeneralPrefs Prefs.
 .
 The root cause of this problem is that src/cc_queue.c includes system headers
 *before* local header files, thus _GNU_SOURCE will not be set when processing
 regex.h. In src/board.c, however, local header files are processed first,
 resulting in different expansions of global.h.
 .
 Best,
 Michael
Author: Michael Tautschnig <mt@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=702439
Last-Update: 2014-10-24

--- a/src/cc_queue.c
+++ b/src/cc_queue.c
@@ -1,7 +1,7 @@
-#include <sys/types.h>
-#include <sys/wait.h>
 #include "coincoin.h"
 #include "site.h"
+#include <sys/types.h>
+#include <sys/wait.h>
 #include <libintl.h>
 #define _(String) gettext (String)
 
